README.md

```diff
@@ -1 +1,61 @@
-# oop-school-library
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# School library 
+
+> A school library app is a tool one can use to record what books are available in the library and who borrows them. This app will enable a user to do the following:
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a book and when.
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- Ruby
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+```
+Ruby
+```
+
+### Setup
+```
+git clone git@github.com:Mithi-code/oop-school-library.git
+
+```
+
+
+## Author
+
+👤 **Mithlesh**
+
+- GitHub: [@githubhandle](https://github.com/Mithi-code)
```

Comment:
- [OPTIONAL] As requested previously, it will be nice to update the Author's(your) information as this will make your `readme file` look more professional and informative.